### PR TITLE
TOML config documentation: general

### PR DESCRIPTION
**Pull request by chrzaszcz**
Document the 'general' section

Functional changes either fixing bugs or simplifying the configuration.
See commit messages for more details and proposed documentation guidelines.

### CODE REVIEW

**Review by NelsonVides — COMMENTED** [review]
I'm honestly very happy with how this is looking 🙂 
Just a couple of language points to say, otherwise I'd approve 👍

File: doc/advanced-configuration/general.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+PostgreSQL's internal structure can make row counting slow.
+Enabling this option uses an alternative query instead of `SELECT COUNT`, that might be not as accurate, but is always fast.
+
+# Access management
+
+User access rules and groups are configured mainly in the `acl`, `shaper` and `access` sections. Here you can find some additional options.
+
+## `mongooseimctl_access_commands`
+* **Scope:** local
+* **Syntax:** TOML table, which **keys** are access groups defined in the `acl` config section and **values** specify allowed administration commands. Each value is a table with the following nested options:
```
[the remark this answers is not in the document]
Comment (chrzaszcz) [reply doc/advanced-configuration/general.md:71]: I made a mistake here, but I know why, see https://thewritepractice.com/whose-vs-which-inanimate-objects/
However, at the moment I am not sure how to rephrase it and I will probably just follow your suggestion.

**Review by NelsonVides — APPROVED** [review]
I'm happy with how this looks, but I'd be good to have more reviews. Let's try to get more people to have a look, and then we'll discuss when we can get it merged :)

**Review by janciesla8818 — CHANGES_REQUESTED** [review]
I really like the approach and division into sections. I see that with the configuration format changed our documentation will gain as well. Really good stuff! Thanks for doing this.

I only have one comment as the `Served hostnames` in `Advanced Configuration` is left without any contents.

**Review by janciesla8818 — APPROVED** [review]
Looks good.

**Pull request merged**